platform/broadcom/docker-syncd-brcm/Dockerfile.j2

```diff
@@ -15,8 +15,24 @@ debs/{{ deb }}{{' '}}
 {%- endfor -%}
 debs/
 
+# Install dependencies for libsaibcm debian packages
+{%- set syncd_brcm_debs = [] %}
+{% for deb in docker_syncd_brcm_debs.split(' ') -%}
+{%- if 'libsaibcm' in deb %}
+RUN apt-get install -y $(dpkg --info debs/{{ deb }} | grep Depends | sed "s/.*ends:\ //" | tr '\n' ' ' | tr ',' ' ')
```

Comment:
i do not know how this can work. you have no access to the internet.

Reply:
Since this is used during build, I assumed we have internet access. Is that not the case? 
Currently I see the following being downloaded during the build. 

```
Unpacking libsaibcm (4.3.0.10-2) ...
^[[91mdpkg: dependency problems prevent configuration of libsaibcm:
 libsaibcm depends on libprotobuf-dev; however:
  Package libprotobuf-dev is not installed.
 libsaibcm depends on libprotobuf-lite17; however:
  Package libprotobuf-lite17 is not installed.
 libsaibcm depends on libprotobuf17; however:
  Package libprotobuf17 is not installed.
 libsaibcm depends on libyaml-dev; however:
  Package libyaml-dev is not installed.

dpkg: error processing package libsaibcm (--install):
 dependency problems - leaving unconfigured
^[[0mProcessing triggers for libc-bin (2.28-10) ...
^[[91mErrors were encountered while processing:
 libsaibcm
^[[0mReading package lists...
Building dependency tree...
Reading state information...
Correcting dependencies... Done
The following additional packages will be installed:
  libc-dev-bin libc6-dev libprotobuf-dev libprotobuf-lite17 libprotobuf17
  libyaml-0-2 libyaml-dev linux-libc-dev zlib1g-dev
Suggested packages:
  glibc-doc manpages-dev libyaml-doc
Recommended packages:
  manpages manpages-dev
The following NEW packages will be installed:
  libc-dev-bin libc6-dev libprotobuf-dev libprotobuf-lite17 libprotobuf17
  libyaml-0-2 libyaml-dev linux-libc-dev zlib1g-dev
0 upgraded, 9 newly installed, 0 to remove and 3 not upgraded.
1 not fully installed or removed.
Need to get 6826 kB of archives.
After this operation, 39.9 MB of additional disk space will be used.
Get:1 http://debian-archive.trafficmanager.net/debian buster/main amd64 libc-dev-bin amd64 2.28-10 [275 kB]
Get:2 http://debian-archive.trafficmanager.net/debian-security buster/updates/main amd64 linux-libc-dev amd64 4.19.171-2 [1430 kB]
Get:3 http://debian-archive.trafficmanager.net/debian buster/main amd64 libc6-dev amd64 2.28-10 [2691 kB]
Get:4 http://debian-archive.trafficmanager.net/debian buster/main amd64 zlib1g-dev amd64 1:1.2.11.dfsg-1 [214 kB]
Get:5 http://debian-archive.trafficmanager.net/debian buster/main amd64 libprotobuf17 amd64 3.6.1.3-2 [814 kB]
Get:6 http://debian-archive.trafficmanager.net/debian buster/main amd64 libprotobuf-lite17 amd64 3.6.1.3-2 [167 kB]
Get:7 http://debian-archive.trafficmanager.net/debian buster/main amd64 libprotobuf-dev amd64 3.6.1.3-2 [1130 kB]
Get:8 http://debian-archive.trafficmanager.net/debian buster/main amd64 libyaml-0-2 amd64 0.2.1-1 [47.2 kB]
Get:9 http://debian-archive.trafficmanager.net/debian buster/main amd64 libyaml-dev amd64 0.2.1-1 [57.4 kB]
Fetched 6826 kB in 1s (9038 kB/s)
```